rpcs3/Emu/RSX/GL/GLPresent.cpp

```diff
@@ -181,7 +181,7 @@ void GLGSRender::flip(const rsx::display_flip_info_t& info)
 		const auto image_to_flip_ = get_present_source(&present_info, avconfig);
 		image_to_flip = image_to_flip_->id();
 
-		if (avconfig._3d) [[unlikely]]
+		if (avconfig.stereo_mode == stereo_render_mode_options::disabled) [[unlikely]]
```

Comment:
I think this is a typo
```suggestion
		if (avconfig.stereo_mode != stereo_render_mode_options::disabled) [[unlikely]]
```